release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -119,6 +119,25 @@ include::snippets/olmv1-tp-extension-support.adoc[]
 
 include::snippets/olmv1-operator-conditions-support.adoc[]
 
+[id="ocp-4-18-extensions-catalog-selection_{context}"]
+==== Improved catalog selection in {olmv1}
+
+With this release, you can perform the following actions to control the selection of catalog content when you install or update a cluster extension:
+
+* Specify labels to select the catalog
+* Use match expressions to filter across catalogs
+* Set catalog priority
+
+For more information, see `Catalog content resolution`.
+
+// Will update with a live link after https://github.com/openshift/openshift-docs/pull/88394 merges
+// xref :../extensions/catalogs/catalog-content-resolution.adoc#catalog-content-resolution[Catalog content resolution].
+
+[id="ocp-4-18-extensions-proxy-and-trustedCA_{context}"]
+==== Basic support for proxied environments and trustedCA certificates
+
+With this release, Operator Controller and catalogd can now run in proxied environments and include basic support for `trustedCA` certificates.
```

Comment:
Not sure myself, but just wondering whether these should be "trusted CA", no markup?